optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/main/resources/org/optaplanner/workbench/screens/solver/backend/server/solution/optacloud.xml

```diff
@@ -1,48 +1,48 @@
-<opta.optacloud.CloudSolution id="1">
+<optacloud.optacloud.CloudSolution id="1">
```

Comment:
Added a comment explaining this file is used in 'Smoke test' tool located in the Solver Editor